README.md

```diff
@@ -1,4 +1,4 @@
-## ent - An Entity Framework For Go
+# ent - An Entity Framework For Go
```

Comment:
Please revert.

Reply:
[MD041 - First line in a file should be a top-level heading](https://github.com/DavidAnson/markdownlint/blob/v0.31.1/doc/md041.md#md041---first-line-in-a-file-should-be-a-top-level-heading)

Reply:
If you don't approve this standard, I will revert it.